### ISSUE TITLE
Fix to apply CSS selector :focus-visible in the current Button

### ISSUE THREAD

**Issue by ManoloAlvarezForo** [cision/rover-ui#347]
Button should be only highlights focused states on keyboard navigation, currently is visible when is clicked as well.
